subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationConfiguration.kt

```diff
@@ -66,6 +66,8 @@ open class InstrumentationConfiguration(val name: String) {
     }
 
     fun data(parentInstrumentationParameters: InstrumentationParameters): Data {
+        require(kubernetesNamespace.isNotBlank())
```

Comment:
`InstrumentationConfiguration` has special `fun validate()`, can this check go there?